shims/npm

```diff
@@ -149,4 +195,17 @@ wrap_npm_if_reshim_is_needed() {
   fi
 }
 
+wrap_npx_if_reshim_is_needed() {
```

Comment:
I don't see this function being called anywhere, should it be called somewhere else? Also, I see that the implementation is very similar to `wrap_npm_if_reshim_is_needed`, maybe we could rewrite the other function to wrap both `npm` and `npx`, wdyt?